proposal.rst

```diff
@@ -239,7 +301,82 @@ Project Timeline
 .. any university classes or exams, vacations, etc), make sure you include them
 .. in your project timeline.
 
-FILLME
+MONTH 1  : 
+
+BIG TASK NAME : DOCUMENTATION STEP
+
+Week 1 
+	- Read and understand code-base further. 
+	- Create a new VagrantFile and make sure that it installs all the xapian dependencies. I have a doc where I have all commands which helped me with my Xapian setup.
+	- Update the "Installation" page with vagrant walk through tutorial.
+
+Week 2 
+	- Re-work on "Practical Example" tutorial to make it more user-friendly. 
```

Comment:
What sort of changes do you feel would make it better? (There is always room for improvement — what I'm asking is what aspects you think are unclear, and how you'd tackle them.)

Reply:
Hey Jaylett, So on the "Introductions", I would recommend to merge the some parts from xapian-core/HACKING such as instructions on MacOSX and linux commands . Currently on Introductions page, there is not much code for demonstrating installation. Also for language specific bindings, I can borrow and take parts from xapian-bindings/INSTALL into main xapian docs installation page.

I found "Practical example" very good to follow, however I was more concerned about the "Core Concepts". For instance, here https://getting-started-with-xapian.readthedocs.io/en/latest/concepts/indexing/databases.html OR terms.html, there is lack of cli code for demonstration. I would definitely include some python examples for this. I understood the value concept but no know how it works in practice : https://getting-started-with-xapian.readthedocs.io/en/latest/concepts/indexing/values.html. So these are mere examples.

Reply:
The instructions in `xapian-core/HACKING` are for people fetching the code from git with the intention of doing development work on Xapian itself, not for people installing releases of Xapian and just wanting to use it.  The instructions for installing releases are in `xapian-core/INSTALL`.  These are aimed at different audiences, and the former instructions are significantly more complicated so it's not helpful to mix them.

But most people just wanting to use Xapian would not build from source but instead install a package of it through their distribution's package manager.